src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1093,6 +1093,11 @@ private static TextSpan GetBlockBodySpan(BlockSyntax body)
                 node = parent;
             }
 
+            if (node is VarPatternSyntax)
```

Comment:
I could check for `token` to be `VarKeyword` instead of `VarPatternSyntax`. Not sure if it matters, or if there is a better third option.

Reply:
This is fine